packages-ts/gauntlet-terra-contracts/src/lib/args.ts

```diff
@@ -2,7 +2,4 @@ export const defaultFlags = {
   delta: 'delta.json',
```

Comment:
Please add a comment that these default flags are not working as expected and reference the GH Issue.

Reply:
Also, please comment about the `RDD` env workaround because of it.